CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## Unreleased
+
+#### Changed
+- Minor update to ensure that for 401 and 403 errors, Restivus sends the response content after a delay.
+#### Fixed
```

Comment:
You can remove the "Changed" section and keep what you have in the "Fixed" section.